### ISSUE TITLE
Switch model

### ISSUE THREAD

**Issue by jrhe** [mikeholscher/zmk-config-mikefive#1]
Would it be possible to provide the switch model? It's defined in the footprint as "PG1316S--装配体.STEP"

**Comment by jedifindtrick** [comment]
@jrhe Just in case you haven't seen it, the model and other stuff should be available now.

**Comment by anscarlett** [comment]
Part is available on JLCPCB, but there's no footprint as yet. Would it be acceptable to upload the files from here?

**Comment by mikeholscher** [comment]
@anscarlett Yes, that's fine, this is the exact footprint as shown on the Kailh specsheet. 

One remark is that we found out the locations of the footprint contact pads as shown on specsheet are very strange in relation to where the actual switch contacts are. One of the center footprint pads is only half on the switch contact. We are currently testing a new footprint where the footprint pads are perfectly aligned with the actual switch contact. I will upload that one on this Github when it is confirmed to work well.

**Comment by anscarlett** [comment]
@mikeholscher Awesome, i'll look out for it. Thanks for your work:)
